triplestore/src/triplestore/exceptions.py

```diff
@@ -8,3 +8,11 @@ class TriplestoreError(Exception):
 
 class BackendNotFoundError(TriplestoreError):
     """Raised when a backend is not registered or supported."""
+
+
+class BackendNotInstalledError(TriplestoreError):

```

Comment:
Shouldn't this (and others) also inherit from `ValueError`, as it seems they are only raised when a wrong argument is passed?